docs: Cli windows installation (#8589)
argoproj/argo-cd: docs/getting_started/install.md

```

This will install the current stable version of Argo CD into the `argocd`
namespace of your Kubernetes cluster. Then continue reading [the first steps](getting_started/first_steps/).

## Production-Grade Installation

When you are ready to use Argo CD in a production environment you have
several installation options.

For a production setup you should pay attention to extra aspects of your installation such as:

* defining the correct resource limits
* applying proper security constraints
* setting up high availability
* managing Argo CD upgrades

> * monitoring Argo CD itself 

Reviewer: Going forward, we should probably fill this with links to the respective sections in the docs

Author: Agreed

Please direct all questions or problems you face using the Helm chart directly
to the chart's maintainers.

## Alternative installation methods

> You can also an infrastructure tool such as [Terraform](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs), [Pulumi](https://www.pulumi.com/registry/packages/kubernetes/) and [Crossplane](https://github.com/crossplane-contrib/provider-kubernetes) to manage your Argo CD installation.

Reviewer: ```suggestion
You can also use infrastructure tools such as [Terraform](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs), [Pulumi](https://www.pulumi.com/registry/packages/kubernetes/) or [Crossplane](https://github.com/crossplane-contrib/provider-kubernetes) to manage your Argo CD installation.
```
